docs/content/tutorial/backups.md

```diff
@@ -150,7 +150,7 @@ You can then make the following changes to the files in the `kustomize/s3` direc
 [Postgres Operator examples](https://github.com/CrunchyData/postgres-operator-examples/fork) repository:
 
 1\. Add the `s3` section to the spec in `kustomize/s3/postgres.yaml` as discussed in the 
-`Using S3 Credentials` section. In addition to that, add the required `eks.amazonaws.com/role-arn` 
+`Using S3 Credentials` [section above](#using-s3-credentials). In addition to that, add the required `eks.amazonaws.com/role-arn` 
```

Comment:
🤔 I interpret the monospace formatting of backticks as code that should be taken literally. Having it on "Using S3 Credentials" is not what I would expect. It's literal, quotes of some kind make sense, so it's not confusing exactly.

🤔 A link on those words makes more sense to me, but the basis is entirely subjective. Do quotes belong around something like this? 🤷🏻 I'd love for us to use technical writing guidelines for things like this.

```suggestion
[Using S3 Credentials](#using-s3-credentials) section above. In addition to that, add the required `eks.amazonaws.com/role-arn` 
```

Reply:
Whoops, sorry about that, missed this when merging, hold on.